ci-operator/step-registry/ipi/conf/nutanix/manual-creds/ipi-conf-nutanix-manual-creds-commands.sh

```diff
@@ -6,17 +6,35 @@ set -o pipefail
 
 source "${SHARED_DIR}/nutanix_context.sh"
 
-credobj="[{\"type\":\"basic_auth\",\"data\":{\"prismCentral\":{\"username\":\"${NUTANIX_USERNAME}\",\"password\":\"${NUTANIX_PASSWORD}\"},\"prismElements\":null}}]"
-credentials=$( echo -n "${credobj}" | base64 -w 0 )
-
-# machine-api credentials manifest
-cat > "${SHARED_DIR}/manifest_openshift-machine-api-nutanix-credentials-credentials.yaml" << EOF
-apiVersion: v1
-kind: Secret
-metadata:
-   name: nutanix-credentials
-   namespace: openshift-machine-api
-type: Opaque
-data:
-  credentials: ${credentials}
+CR_DIR="/tmp/credentials_request"
+mkdir -p "${CR_DIR}"
+
+MANIFEST_PREFIX="${SHARED_DIR}/manifests"
+
+# Create the Nutanix credentials secret
+cat > ${SHARED_DIR}/credentials <<EOF
+credentials:
+- type: basic_auth
+  data:
+    prismCentral:
+      username: ${NUTANIX_USERNAME}
+      password: ${NUTANIX_PASSWORD}
+    prismElements: null
 EOF
+
+# Extract credential requests
+oc registry login
+oc adm release extract --credentials-requests --cloud=nutanix --to "${CR_DIR}" "${RELEASE_IMAGE_LATEST}"
+
+# Create the credentials request manifest
+ccoctl nutanix create-shared-secrets \
+      --credentials-requests-dir="${CR_DIR}" \
+      --output_dir="/tmp" \
+      --credentials-source-filepath="${SHARED_DIR}/credentials"
+
+# Copy the manifest to the shared directory
+pushd /tmp
+for FILE in *.yaml; do
+  cp "${FILE}" "${MANIFEST_PREFIX}_${FILE}"
```

Comment:
I think it should be:

cp "${FILE}" "${MANIFEST_PREFIX}/${FILE}"

Reply:
No, it shouldn't.

Please take a look https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L364 that handle manifests before installation.

Reply:
Okay, got it.